moped-editor/package.json

```diff
@@ -9,11 +9,10 @@
     "start:local": "export HTTPS=true && react-scripts start",
     "start:staging": "env-cmd -e staging npm run start:local",
     "start:production": "env-cmd -e production npm run start:local",
-    "build": "env-cmd -e local npm run build:local",
+    "build": "react-scripts build",
```

Comment:
Is the command not working?

Reply:
Nice catch, Sergio! I didn't see that update. Not sure when this happened but maybe when I added `axios`. Looks like there were two build commands before. I'll try to run it and make sure everything looks okay.